src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java

```diff
@@ -1225,6 +1225,11 @@ private ElementFacade getDeleteCommentLabel(String comment) {
                                           comment));
   }
 
+  private ElementFacade getCancelKudosLabel(String comment) {
+    return findByXPathOrCSS(String.format("//*[contains(text(),'%s')]/preceding::*[@class='v-list-item__title pl-3' and contains(text(),'Cancel kudos sending')]",
```

Comment:
please avoid using `@class=` and use `contains(@class, '...')` instead + avoid using `preceding` (too vague) and use `/parent::*//*....` instead or `preceding-sibling`